.circleci/config.yml

```diff
@@ -48,7 +48,11 @@ jobs:
           steps:
             - run:
                 name: install cbor-diag
-                command: cargo install --force cbor-diag-cli
+                command: |
```

Comment:
`cbor-diag-cli` no longer builds using Rust < 1.81 because of some nested dependency issue.

I created a binary release and used it here.